@here/harp-mapview/lib/DecodedTileHelpers.ts

```diff
@@ -646,15 +646,14 @@ export function applyBaseColorToMaterial(
 
     opacity = THREE.Math.clamp(opacity, 0, 1);
     material.opacity = opacity;
+    materialColor.setRGB(r, g, b);
 
     const opaque = opacity >= 1.0;
     if (!opaque) {
-        enableBlending(material);
+        enableBlending(material, true);
```

Comment:
Just wondering if I should expose this flag is method as optional, for example:
```
export function applyBaseColorToMaterial(
    material: THREE.Material,
    materialColor: THREE.Color,
    technique: Technique,
    techniqueColor: Value,
    isUpdate?: boolean
    zoomLevel?: number
   )
```
Then we could manipulate if material update comes from frame update or during initialization phase. 
Currently as I see applyBaseColorToMaterial is used in both scenarios, see:
- DecodedTileHelpers,
- TileGeometryCreator.

for me it seems that both cases may force material.needsUpdate because in DecodedTileHelpers material will be invalidated anyway (true?) and in TileGeometryCreator in case of opacity change (interpolation) we need to force material update.